src/plugins/router.js

```diff
@@ -0,0 +1,14 @@
+import { health } from '../routes/health.js'
+import { example } from '../routes/example.js'
+
+export const router = {
+  plugin: {
+    name: 'router',
+    async register (server) {
+      // Application specific routes, add your own routes here.
+      const appSpecificRoutes = [example]
+
+      server.route([health].concat(appSpecificRoutes))
+    }
+  }
+}
```

Comment:
Majority of Defra repos have a dedicated plugin directory for plugins, rather than embedding them across logic.

Reply:
Yeah this a pattern I have come to notice with Hapi. Sounds sensible